### PR TITLE
Updates from rspec-dev (2020-11-18)

### PR DESCRIPTION
**Pull request by JonRowe**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,42 +0,0 @@
-# This file was generated on 2020-11-16T22:10:29+00:00 from the rspec-dev repo.
```
Comment (pirj) [inline appveyor.yml:1]: ❤️

**Review by pirj — APPROVED** [review]
Awesome!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -40,3 +40,30 @@ jobs:
...
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler: 2
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: cinst ansicon
+      - run: bundle exec rspec --backtrace
+        env:
+          APPVEYOR: true
```
Comment (benoittgt) [inline .github/workflows/ci.yml:69]: Is is still needed?

**Review by benoittgt — APPROVED** [review]
Awesome!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (JonRowe) [reply .github/workflows/ci.yml]: It was until support was merged changing the checks.

**Comment by JonRowe** [comment]
Merging because all GHA green 🥳

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -97,6 +97,16 @@ def handle_current_dir_change
     with_env_vars('SHELL' => '/usr/local/bin/bash', &ex)
   end
 
+  if ENV['CI'] && RSpec::Support::OS.windows? && RUBY_VERSION.to_f < 2.3
+    c.around(:example, :emits_warning_on_windows_on_old_ruby) do |ex|
+      ignoring_warnings(&ex)
+    end
+
+    c.define_derived_metadata(:pending_on_windows_old_ruby => true) do |metadata|
```
Comment (benoittgt) [inline spec/spec_helper.rb:105]: Nice! 👌

**Comment by benoittgt** [comment]
Congrats!!

**Pull request merged**